src/platform/backends/libvirt/libvirt_virtual_machine.cpp

```diff
@@ -278,7 +278,26 @@ void mp::LibVirtVirtualMachine::start()
     update_state();
 
     if (virDomainCreate(domain.get()) == -1)
-        throw std::runtime_error(virGetLastErrorMessage());
+    {
+        state = State::suspended;
+        update_state();
+
+        std::string error_string{virGetLastErrorMessage()};
+        if (error_string.find("virtio-net-pci.rom: 0x80000 in != 0x40000") != std::string::npos)
+        {
+            error_string = fmt::format("Unable to start suspended instance due to incompatible save image.\n"
+                                       "Please use the following steps to recover:\n"
+                                       "  1. snap revert multipass\n"
```

Comment:
Definitely less likely case, but possible user has >1 multipass snap updates between creation of the old save image, and getting this error. Can we specify a particular snap revision/channel?

Reply:
Yeah that was my only concern indeed. We can't specify a revision, but we can maintain a `core16` channel for a while.

Reply:
Ok, right.  What would the `snap` command be for something like this?

Reply:
`snap refresh multipass --channel core16/beta` and then `snap refresh multipass --channel beta`.

Reply:
@Saviq, cool, thanks.  I'll change this to reflect that.

Reply:
@Saviq, for the `core16` fallback, do we care about users on the `edge` channel?

Reply:
I think they'll know what to do (and no, I don't think we'll have `core16/edge`).

We'll actually move this to `--stable` once we have one.

Reply:
Ok, changed the `snap` instructions.